algorand/token_bridge.py

```diff
@@ -522,8 +522,8 @@ def completeTransfer():
                     tidx.store(Txn.group_index() + Int(1)),
                     MagicAssert(And(
                         Gtxn[tidx.load()].type_enum() == TxnType.ApplicationCall,
-                        Gtxn[tidx.load()].application_args[0] == Txn.application_args[0],
-                        Gtxn[tidx.load()].application_args[1] == Txn.application_args[1],
+                        Gtxn[tidx.load()].application_args[0] == Bytes("base16", "0x903f4535"), # sha256("portal_transfer(byte[])byte[]")[:4]
```

Comment:
This is the same magic number as above.  Would life be easier for you if it was #defined?